common/src/View/GameDialog.cpp

```diff
@@ -186,14 +185,21 @@ namespace TrenchBroom {
             mapFormatLayout->addWidget(label, 0, Qt::AlignRight | Qt::AlignVCenter);
             mapFormatLayout->addWidget(m_mapFormatComboBox, 1, Qt::AlignLeft | Qt::AlignVCenter);
 
+            auto* mapFormatWidget = new QWidget();
+            mapFormatWidget->setLayout(mapFormatLayout);
+
             auto* outerSizer = new QVBoxLayout();
             outerSizer->setContentsMargins(QMargins());
             outerSizer->setSpacing(0);
             outerSizer->addWidget(m_gameListBox, 1);
             outerSizer->addWidget(new BorderLine(BorderLine::Direction::Horizontal), 1);
-            outerSizer->addLayout(mapFormatLayout);
+            outerSizer->addWidget(mapFormatWidget);
             panel->setLayout(outerSizer);
 
+            if (!formatPrompt) {
```

Comment:
Instead of hiding the list of supported formats, I think we should add an entry "Autodetect" and preselect that. This way, a user can override the auto detection and try a specific format (e.g. for debugging an issue with their map files).

Reply:
I'm having trouble imagining a concrete use case where this is helpful debugging, maybe it's easier to discuss the details here on discord

Reply:
As discussed below, IMO this is necessary for the case where multiple formats are possible, and the user does not want the first one that works.

Reply:
Re debugging: This comment was made before I suggested that we need to show all of the error messages and not only the last one. So for debugging it is less relevant, but I still think it's important for the user use case.